src/gluonts/mx/model/forecast.py

```diff
@@ -61,6 +61,7 @@ def __init__(
         freq: str,
         item_id: Optional[str] = None,
         info: Optional[Dict] = None,
+        index: Optional[Union[List[pd.Timestamp], pd.DatetimeIndex]] = None,
```

Comment:
We could create a type-alias for this.

Reply:
so this is was initial thought... but i believe i only need the index to be `pd.DatetimeIndex`...